### PR TITLE
feat: add angular js support for icons

### PR DESCRIPTION
**Pull request by domhorvathtw**
### PR to merge to `beta` branch to create prerelease
**Version `v2.0.0-beta.2`**

## ❓ Context
TransferWise is still heavily using AngularJS in legacy codebases. In order to replace our icons everywhere on the web, we need to support these legacy environments.

## 🚀 Changes
- Refactor build script and add angular js components templates to generate files the same way as we do it for react components.
- Update rollup config to add angular bundle config

## 💬 Considerations
The end result is a **UMD bundle** that has the angularJs module and includes all the icons.

To use it in your angularJs app, just import the module 
```js
import { TwIconsModule } from "@transferwise/icons/lib/angular";

angular.module("app", [TwIconsModule]);
```

and in your angularJs template use the icons
```html
<tw-activity-icon size="16" filled="true"></tw-activity-icon>
<tw-alert-circle-icon size="24"></tw-alert-circle-icon>
```

## ✅ Checklist

- [x] The package version is bumped according to README in `package.json`, `package-lock.json`, and `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+import { TwIconsModule } from "@transferwise/icons/lib/angular";
+
+angular.module("app", [TwIconsModule]);
+```
+
+and in your angularJs template use the icons
+```html
+<tw-activity-icon size="16" filled="true"></tw-activity-icon>
+<tw-alert-circle-icon size="24"></tw-alert-circle-icon>
+```
```
Comment (dozortsev) [inline CHANGELOG.md:16]: nice, we will add this part to `README.md` [later](https://transferwise.atlassian.net/browse/DS-367)

File: rollup.config.js [hunk 2]
```diff
@@ -30,3 +30,19 @@ export default {
...
+  output: {
+    file: 'lib/angular/index.js',
+    name: 'index.js',
+    format: 'umd',
+    globals: {
+      angular: 'angular',
+    },
+  },
+  external: [...Object.keys(pkg.dependencies || {}), ...Object.keys(pkg.peerDependencies || {})],
+  plugins: [terser(), analyze({ summaryOnly: true })],
```
Comment (dozortsev) [inline rollup.config.js:45]: probably minor but I guess we don't need `terser()` here as it's only for ES modules while here for bundle UMD?

plugin [docs](https://www.npmjs.com/package/rollup-plugin-terser):

> .. Rollup plugin to minify generated es bundle. Uses terser under the hood.

and if you further check terser [docs](https://github.com/terser/terser) it also says same.

Comment (domhorvathtw) [reply rollup.config.js:45]: it minifies the **UMD** bundle as well. I tested the result with and without the terser plugin

Comment (dozortsev) [reply rollup.config.js:45]: 🤔 okey...

**Pull request merged**